packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts

```diff
@@ -0,0 +1,251 @@
+
+const runQuery = async (query: string) => {
+  try {
+    const q = [query, ...fragments].join('\n');
+    const response = await GRAPHQL_CLIENT.query(q, {});
+    return response;
+  } catch (e) {
+    console.error(e);
+    return null;
+  }
+};
+
+const createEntries = async () => {
+  // create posts
+  await runQuery(getCreatePostsMutation('testuser', 'test', 157, 10, 97.4, true));
+  await runQuery(getCreatePostsMutation('testuser', 'test title', 60, 30, 21.0, false));
+  await runQuery(getCreatePostsMutation('sampleuser', 'test title', 160, 30, 97.6, false));
+  await runQuery(getCreatePostsMutation('sampleuser', 'test TITLE', 170, 30, 88.8, true));
+  await runQuery(getCreatePostsMutation('testuser', 'test title', 200, 50, 11.9, false));
+  await runQuery(getCreatePostsMutation('testuser', 'test title', 170, 30, 88.8, true));
+  await runQuery(getCreatePostsMutation('testuser', 'test title', 160, 30, 97.6, false));
+  await runQuery(getCreatePostsMutation('testuser', 'test title', 170, 30, 77.7, true));
+  // Waiting for the ES Cluster + Streaming Lambda infra to be setup
+  await cf.wait(120, () => Promise.resolve());
+};
+
+beforeAll(async () => {
+  const validSchema = `
+    type Post @model @searchable {
+      id: ID!
+      author: String!
+      title: String
+      content: String
+      url: String
+      ups: Int
+      downs: Int
+      version: Int
+      postedAt: String
+      createdAt: AWSDateTime
+      comments: [String!]
+      ratings: [Int!]
+      percentageUp: Float
+      isPublished: Boolean
+      jsonField: AWSJSON
+    }
+    `;
+  const transformer = new GraphQLTransform({
+    featureFlags,
+    transformers: [
+      new ModelTransformer(), new SearchableModelTransformer(),
+    ],
+  });
+  try {
+    await awsS3Client.createBucket({ Bucket: BUCKET_NAME }).promise();
+  } catch (e) {
+    console.error(`Failed to create bucket: ${e}`);
+  }
+  try {
+    const out = transformer.transform(validSchema);
+    const finishedStack = await deploy(
+      customS3Client,
+      cf,
+      STACK_NAME,
+      out,
+      {},
+      LOCAL_FS_BUILD_DIR,
+      BUCKET_NAME,
+      S3_ROOT_DIR_KEY,
+      BUILD_TIMESTAMP,
+    );
+    // Arbitrary wait to make sure everything is ready.
+    await cf.wait(120, () => Promise.resolve());
+    expect(finishedStack).toBeDefined();
+    const getApiEndpoint = outputValueSelector(ResourceConstants.OUTPUTS.GraphQLAPIEndpointOutput);
+    const getApiKey = outputValueSelector(ResourceConstants.OUTPUTS.GraphQLAPIApiKeyOutput);
+    const endpoint = getApiEndpoint(finishedStack.Outputs);
+    const apiKey = getApiKey(finishedStack.Outputs);
+    expect(apiKey).toBeDefined();
+    expect(endpoint).toBeDefined();
+    GRAPHQL_CLIENT = new GraphQLClient(endpoint, { 'x-api-key': apiKey });
+
+    // Create sample mutations to test search queries
+    await createEntries();
+  } catch (e) {
+    console.error(e);
+    expect(true).toEqual(false);
```

Comment:
I realize that this was copied from other tests, but I think we should just throw the original error.

Reply:
Agree, I will update.